README.md

```diff
@@ -1,7 +1,7 @@
 Orange
 ======
 
-[![Join the chat at https://gitter.im/biolab/orange3](https://badges.gitter.im/biolab/orange3.svg)](https://gitter.im/biolab/orange3?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Discord Chat](https://img.shields.io/discord/308323056592486420.svg)](https://discord.gg/47gRDUQ)  
```

Comment:
`https://img.shields.io/discord/624519803914158080.svg` is the icon URL which should be used.
Remove the double space at the end of the line (newline in MD)